sdk-api-src/content/winbase/nf-winbase-setcurrentdirectory.md

```diff
@@ -72,6 +72,8 @@ By default, the name is limited to MAX_PATH characters.
 
 > [!TIP]
 > Starting with Windows 10, Version 1607, you can opt-in to remove the MAX_PATH limitation. See the "Maximum Path Length Limitation" section of [Naming Files, Paths, and Namespaces](/windows/win32/fileio/naming-a-file) for details.
+>
+> Note that setting a current directory longer than the MAX_PATH limitation will cause <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-createprocessw">CreateProcess</a> to fail unless the caller overrides the current directory.
```

Comment:
Optional suggestion: "overrides the current directory" is a bit ambiguous here. Should we just nix that and leave that as an exercise to the reader?

```suggestion
> Note that setting the current directory to a path longer than the MAX_PATH limitation will cause <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-createprocessw">CreateProcess</a> to fail.
```

Reply:
Yes, I've removed that clause. I didn't know what that meant. If it's important, and if someone can explain it (in a different PR or GitHub issue), then I'll be happy to add it.

Reply:
I just mean that `CreateProcess` allows setting a current directory explicitly and that this will still work. I am however happy to let users infer this for themselves.